src/tests/src/libs/antares/study/parts/hydro/CMakeLists.txt

```
@@ -1,76 +1,66 @@
 # Useful variables definitions
 set(src_libs_antares_study "${CMAKE_SOURCE_DIR}/libs/antares/study")
 set(src_libs_antares "${CMAKE_SOURCE_DIR}/libs/antares")
+set(src_test_utilis "${CMAKE_SOURCE_DIR}/tests/src/utlis")
```

I think there are two typos
```suggestion
set(src_test_utils "${CMAKE_SOURCE_DIR}/tests/src/utils")
```

---

Done!